src/ralph/trade_marks/models.py

```diff
@@ -118,6 +142,11 @@ class TradeMark(Regionalizable, AdminAbsoluteUrlMixin, BaseObject):
         related_name='+',
         through='TradeMarksLinkedDomains',
     )
+    registering_institution = models.ForeignKey(
```

Comment:
Maybe `registrar` would be better? More concise and probably more natural. Google search for registering institution brings up no institution, only instructions on how to register an institution :)

Reply:
Convinced me, I'll change it.